Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/HistoryViewController.swift

```diff
@@ -168,8 +172,8 @@ class HistoryViewController: SiteTableViewController, ToolbarUrlActionsProtocol
             title: Strings.History.historyClearAlertTitle, message: Strings.History.historyClearAlertDescription, preferredStyle: style)
         
         alert.addAction(UIAlertAction(title: Strings.History.historyClearActionTitle, style: .destructive, handler: { _ in
-            DispatchQueue.main.async {
-                Historyv2.deleteAll { [weak self] in
+            DispatchQueue.main.async { [weak self] in
+                self?.historyManager.deleteAll {
```

Comment:
This can be regular `self`, no need for weak (no chance of retain cycle)

Reply:
Removed at https://github.com/brave/brave-ios/pull/4024/commits/41a21e807d1945db65efb90f53dca56e432de591